tap_gorgias/schemas/voice_call_recordings.json

```diff
@@ -0,0 +1,81 @@
+{
+  "type": [
+    "null",
+    "object"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": [
+        "null",
+        "integer"
+      ],
+      "description": "ID of the voicemail or call recording."
+    },
+    "call_id": {
+      "type": [
+        "null",
+        "integer"
+      ],
+      "description": "ID of the voice call the voicemail or call recording belongs to."
+    },
```

Comment:
same here. the id property is the primary key and cannot be null. I would also assume that call_id should never be null